### PR TITLE
Fix documention minor issues

### PR DESCRIPTION
**Pull request by AmirRezaM75**

### CODE REVIEW

File: packages/website/versioned_docs/version-latest/getting-started/typespec-for-openapi-dev.md [hunk 1]
```diff
@@ -113,6 +113,10 @@ But a namespace may contain operations for a set of related paths (depending on
 The http method decorators also accept an explicit path, which is appended to the namespace path.
 
 ```typespec
+model PetId {
+  @path petId: int32;
+}
+
```
Comment (AmirRezaM75) [inline packages/website/versioned_docs/version-latest/getting-started/typespec-for-openapi-dev.md:119]: Model is used in line 126. define it here, make it more clear

File: packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ emit:
   - emitter1
 ```
 
-in `<my-pkg>/tspconfig.yaml`, enable `emitter2` using the options specified in the parent `tspconfig.yaml
+in `<my-pkg>/proj2/tspconfig.yaml`, enable `emitter2` using the options specified in the parent `tspconfig.yaml
```
Comment (AmirRezaM75) [inline packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md:73]: In my opinion the second example is not necessary and has been covered above.

File: packages/website/versioned_docs/version-latest/introduction/style-guide.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ The guidelines in this article are used in TypeSpec Core libraries. You can use
 | scalar           | camelCase                                    | `scalar uuid extends string;`                    |
 | model            | PascalCase                                   | `model Pet {}`                                   |
 | model property   | camelCase                                    | `model Pet {furColor: string}`                   |
-| enum             | PascalCase                                   | `model Pet {furColor: string}`                   |
+| enum             | PascalCase                                   | `enum Direction {}`                              |
 | enum member      | camelCase                                    | `enum Direction {up, down}`                      |
```
Comment (AmirRezaM75) [inline packages/website/versioned_docs/version-latest/introduction/style-guide.md:21]: Although all examples mentioned here https://microsoft.github.io/typespec/language-basics/enums has PascalCase members :confused:

**Comment by AmirRezaM75** [comment]
> @microsoft-github-policy-service agree

**Comment by timotheeguerin** [comment]
> > @microsoft-github-policy-service agree

Hi @AmirRezaM75  thanks for the contribution, It seems like it hasn't picked up the cla agreement, I think it might be because it has the markdown quote prefix `>` 

If you comment again without I think it should work!

File: packages/website/versioned_docs/version-latest/introduction/style-guide.md (hunk 3 above)
Comment (timotheeguerin) [reply packages/website/versioned_docs/version-latest/introduction/style-guide.md:21]: Good catch, sounds like we need to update the other docs too. They were written before we came to an agreement for the style guide

File: packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md (hunk 2 above)
Comment (timotheeguerin) [reply packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md:73]: that's fair, feel free to remove that one

Comment (AmirRezaM75) [reply packages/website/versioned_docs/version-latest/introduction/configuration/configuration.md]: removed

**Comment by AmirRezaM75** [comment]
@microsoft-github-policy-service agree

**Comment by AmirRezaM75** [comment]
@timotheeguerin
I agreed with policy in dedicated comment, tell me if it doesn't work.
Does docs' compiler script runs in your CI/CD or do I have to run it locally and commit changes?
Ready for review.

**Comment by timotheeguerin** [comment]
Seemed to have worked thanks! Yes the docs will get published automatically however I just noticed you updated the “latest” docs version in versioned_docs folder. Those are going to get overwritten next release(next week) by the content of docs/ would you mind making the same fixes there as well

**Comment by AmirRezaM75** [comment]
> Seemed to have worked thanks! Yes the docs will get published automatically however I just noticed you updated the “latest” docs version in versioned_docs folder. Those are going to get overwritten next release(next week) by the content of docs/ would you mind making the same fixes there as well

@timotheeguerin Done.

**Comment by AmirRezaM75** [comment]
> Seemed to have worked thanks! Yes the docs will get published automatically however I just noticed you updated the “latest” docs version in versioned_docs folder. Those are going to get overwritten next release(next week) by the content of docs/ would you mind making the same fixes there as well

@timotheeguerin Done.

**Pull request merged**